README.md

```diff
@@ -15,26 +15,27 @@ See the License for the specific language governing permissions and
 limitations under the License.
 --->
 
-
 # <img src="logos/logo.svg" width="180">
-###A Tool for Complex and Scalable Data Access Policy Enforcement
+### A Tool for Complex and Scalable Data Access Policy Enforcement
 
 Palisade aims to provide a stable framework for defining and enforcing policies for accessing data.
 
 
-####Introduction
-Modern organisations can have multiple datasets that are each managed and accessed according to different policies and procedures. This can make it complicated for analysis while still ensuring reglatory compliance.  Most technologies "out of the box" do not provide the necessary solution for managing these complex issues for data access. This leads to data being siloed and often accesses being restricted with the outcome that the insight from data can be lost.   
+#### Introduction
+Modern organisations can have multiple datasets that are each managed and accessed according to different policies and procedures. This can make it complicated for analysis while still ensuring regulatory compliance.  Most technologies "out of the box" do not provide the necessary solution for managing these complex issues for data access. This leads to access to data being overly restrictive with potential insights from data being lost.   
```

Comment:
As these are markdown files, the general consensus is that it should stylistically be presented inside the file as one sentence per line.
See [this thread](https://github.com/prettier/prettier/issues/3302) discussing pros and cons, the biggest here being git diffs work per line.